cfunits-python/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --record record.txt
```

Comment:
+1 for switching to `cf_units`.  We went through a lot of pain with the others, and `cf_units` has the brightest future, for sure, now that the British Met Office is on board.